src/main/resources/templates/gardens/list.html

```diff
@@ -60,28 +60,27 @@ <h4 class="garden-subtitle">Here are the water gardens that are featured on our
         <th:block th:if="${title.equals('Garden List')}">
             <p>Displaying All Gardens</p>
         </th:block>
-        <!-- hide link if on main gardens/list/ (only show when on search)-->
+    </div>
 
-        <hr>
-        <div th:fragment="garden-card" class="container-fluid">
```

Comment:
It looks like this line being changed is messing with the categories functionality on the home page.

Reply:
Should I set up the garden-card in fragments.html for the front page features? If so do you want me to have these cards pull from there too?